docs/reference/release-notes/8.0.0-alpha1.asciidoc

```diff
@@ -1,44 +1,520 @@
+:es-issue: https://github.com/elastic/elasticsearch/issues/
+:es-pull:  https://github.com/elastic/elasticsearch/pull/
+
```

Comment:
We pull this in from an attributes file in the `elastic/docs` repo. We don't need it here.
```suggestion
```